vendor/vendor.json

```diff
@@ -6,6 +6,12 @@
 			"path": "context",
 			"revision": ""
 		},
+		{
+			"checksumSHA1": "OeMM9kqd+l7qKt2ebPlkuNTj3iM=",
+			"path": "github.com/Thomasdezeeuw/ini",
```

Comment:
This is a new dependency to package. (CC @Conan-Kudo and @mwhudson ) under MIT license.

Reply:
This should be causing Fedora builds to fail, then, since we don't do vendored and I don't see a corresponding change to the spec to request this dependency.